lib/anyway/config.rb

```diff
@@ -373,7 +373,7 @@ def validate_required_attributes!
         values[name].nil? || (values[name].is_a?(String) && values[name].empty?)
       end.then do |missing|
         next if missing.empty?
-        raise_validation_error "The following config parameters are missing or empty: #{missing.join(", ")}"
+        raise_validation_error "The following config parameters for `#{self.class.name}` are missing or empty: #{missing.join(", ")}"
```

Comment:
What about adding `.config_name` to the message as well? Maybe, `#{self.class.name}<config_name:#{self.class.config_name}>` or similar?

Reply:
That is a great point. We always use the default, so I forgot you could even override that. Will tweak that now

Reply:
I went with `Class(config_name: config)` following the convention from ActiveRecord classes